src/parser/HLSTree.cpp

```diff
@@ -423,6 +419,27 @@ bool HLSTree::ParseManifest(std::stringstream& stream)
   return true;
 }
 
+void HLSTree::SaveManifest(AdaptationSet* adp, const std::stringstream& data, std::string_view info)
+{
+  if (m_pathSaveManifest.empty())
+    return;
+
+  std::string fileNameSuffix = "master";
```

Comment:
Does braces initialisation work here? `std::string fileNameSuffix{"master"};`

Reply:
i havent use {} brackets since recent discussions on slack,
so we allow use `=` within code block to make it more readable
but on class/struct vars initialization still use `{}`
see docs changes here: https://github.com/xbmc/xbmc/pull/22639